src/tools/analyzer/modules/request.py

```diff
@@ -214,7 +214,7 @@ def print_formatted_verbose(self, source):
                         print("       - " + name[:-1])
                         cr_done.append(cr)
                 if (id and ("UID" in cidline or "GID" in cidline)):
-                    if id not in id_done:
+                    if id not in id_done and bool(re.search(r'\d', id)):
```

Comment:
You have in the commit message "ensuring the parsed field is a digit" but this regex just checks if there is any digit so `id`  "abcd3xyz" will be evaluated as `True`. Is that OK? Shouldn't we use something more strict like `r'^\d+$'` ?

Reply:
This was intentional, because the correct `list -v` output contains non-digit characters as well, for example: `GID:1031401119@testrealm.test`

This PR is intended to filter out the line ` - cache` below, but I wanted to make it more generic than just looking for the word 'cache'
~~~
2022-11-15 12:02:18: [uid 0] CID #1: id
   - User by name
       - aduser1@domain-zflo.com
   - Group by ID
       - GID:1031401119@testrealm.test
       - GID:1031401119@domain-zflo.com
       - cache
   - Initgroups by name
       - aduser1@domain-zflo.com
   - Group by ID
       - GID:1031401120@testrealm.test
       - GID:1031401120@domain-zflo.com
       - cache
   - Group by ID
       - GID:1031400513@testrealm.test
       - GID:1031400513@domain-zflo.com
       - cache
~~~

Reply:
Thank you for the explanation.

Reply:
Thanks for the review.